praw/models/reddit/subreddit.py

```diff
@@ -1708,12 +1708,28 @@ def __init__(self, subreddit):
         """
         self.subreddit = subreddit
 
-    def comments(self):
+    def comments(self, pause_after=None):
         """Yield new comments as they become available.
 
         Comments are yielded oldest first. Up to 100 historical comments will
         initially be returned.
 
+        It's possible to stop the fetching loop on each failure so the calling
```

Comment:
Removed and tried to make the style more consistent. Not sure whether the first example (which doesn't pass pause_after) adds values. It's simple, which is a positive.